committeeoversightapp/views.py

```diff
@@ -36,8 +38,41 @@ def archive_url(url):
     wayback_host = 'http://web.archive.org'
     save_url = '{0}/save/{1}'.format(wayback_host, url)
     archived = requests.get(save_url)
-    archive_url = '{0}{1}'.format(wayback_host, archived.headers['Content-Location'])
-    return archive_url
+    try:
+        archive_url = '{0}{1}'.format(wayback_host, archived.headers['Content-Location'])
+        return archive_url
+    except KeyError:
+        return None
+
+def save_document(url, note, event):
```

Comment:
I'd put this function (and the others above it) in a `utils.py` file, since they are not technically views.